src/runtime.rs

```diff
@@ -1,17 +1,18 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+#![allow(dead_code)]
+
```

Comment:
Remove the code that ins't used

Reply:
Ok

Reply:
Done

Reply:
Depends on how fast you review the said PR